scipy/linalg/lapack.py

```diff
@@ -113,6 +113,11 @@
    cgehrd_lwork
    zgehrd_lwork
 
+   sgejsv
+   dgejsv
+   cgejsv
+   zgejsv
```

Comment:
@ilayn CircleCI failed because it expected to find {c,z}gejsv but couldn't. I think these will need to be removed also.

```suggestion
   cgejsv
   zgejsv
```